src/Deedle/Stats.fs

```diff
@@ -628,6 +628,29 @@ type Stats =
       let b = quickSelectInplace (mid - 1) values
       (a + b) / 2.0
 
+  /// Returns the series of main statistic values of the series.
+  ///
+  /// [category:Series statistics]
+  static member inline describe (series:Series<'K, 'V>) = 
+    match series with
```

Comment:
I'm getting a warning here saying that this will make `'V` a `float`:

> warning FS0064: This construct causes code to be less generic than indicated by the type annotations. The type variable 'V has been constrained to be type 'float'.

You can fix that by adding `box`, but then you also need to specify the type argument for the series key in the pattern matching:

    match box series with
    | :? Series<'K, float> as floatSeries -> 

Also, I think the returned series should probably not be `Series<string, float option>` but just `Series<string, float>` because series has missing values built-in. To do that, you can replace `Series.ofObservations` with `Series.ofOptionalObservations`

Reply:
@tpetricek I already refactored it in https://github.com/fslaborg/Deedle/commit/6ba06fd0c7652495c252c16707aa8a896e0c61ef. There's no longer warning. It will output Series<string, float>.

Reply:
👍 Sorry, I missed that! Sounds good.